README.md

```diff
@@ -37,6 +37,7 @@ The pipeline then:
 * predicts protein-coding genes for the assemblies using [Prodigal](https://github.com/hyattpd/Prodigal)
 * performs metagenome binning using [MetaBAT2](https://bitbucket.org/berkeleylab/metabat/src/master/), and checks the quality of the genome bins using [Busco](https://busco.ezlab.org/)
 * assigns taxonomy to bins using [GTDB-Tk](https://github.com/Ecogenomics/GTDBTk) and/or [CAT](https://github.com/dutilh/CAT)
+* performs (optionaly) ancient DNA assembly validation using [PyDamage](https://github.com/maxibor/pydamage) and contig consensus sequence recalling with [Freebayes](https://github.com/freebayes/freebayes) and [BCFtools](http://samtools.github.io/bcftools/bcftools.html).
```

Comment:
I'd rather put that after assembly, similar to the figure

Reply:
Done